zulipterminal/config/keys.py

```diff
@@ -373,6 +374,16 @@ class KeyBinding(TypedDict, total=False):
     ]
 )
 
+STD_NAV_CMDS = [
```

Comment:
While it is shorter to type, STD is not a well-known shorthand. How about `BASIC_NAV_COMMANDS`/`PRIMARY_NAV_COMMANDS`?

Reply:
Maybe `BASIC` is a better option that `PRIMARY` as it might be confused with primary keys. I'll change this.